webapp/src/ts/services/xml-forms.service.ts

```diff
@@ -264,11 +268,14 @@ export class XmlFormsService {
           // fallback for backwards compatibility
           return this.getByView(internalId);
         }
+        console.error(`Error in XMLFormService getByView : the form "${internalId}" was not found`);
```

Comment:
This error is triggered in the `get` function, not `getByView` like the message suggests: 
```suggestion
        console.error(`Error in XMLFormService.get : the form "${internalId}" was not found`);
```